seed/client/src/main/scala/example/seed/client/process/ProtoPeopleServiceClient.scala

```diff
@@ -54,7 +54,7 @@ object ProtoPeopleServiceClient {
           _      <- L.info(s"")
           result <- client.getPerson(PeopleRequest(name))
           _      <- L.info(s"$serviceName - Request: $name - Result: $result")
-        } yield result.person.get
+        } yield result.person.getOrElse(Person("default", 0))
```

Comment:
I think it'd fine as is, but this kind of issue should be transformed into something more meaningful thing, like a `PersonNotFound` type error or something like that. The same applies for the rest of the cases. This is just an example and we don't need to do a sophisticated API here, but I wanted to bring this thought here :)

Reply:
ah that's a great thought; yeah, tbh I wasn't really sure what we wanted the behavior to be since I only had to make the changes because proto3 makes its content optional (which makes sense, it's just necessitated a bigger refactor).  

Honestly, I'd be game to make that change!  I'd love for these examples to be exemplary as they can and I just finished some client work yesterday so I've got time today :)

Reply:
👍 It doesn't need to be part of this PR :)

Reply:
oooops, just did it ;)

Reply:
@juanpedromoreno let me know if my custom error approach looks okay (no hurry though 😄 )